spec/integration/alerts_spec.rb

```diff
@@ -475,11 +557,38 @@ module Hawkular::Alerts::RSpec
     end
   end
 
-  describe 'Alert/Events', :vcr do
-    VCR.configure do |c|
-      c.default_cassette_options = {
-        match_requests_on: [:method, VCR.request_matchers.uri_without_params(:startTime, :endTime)]
-      }
+  describe 'Alert/Events' do
+    before(:all) do
+      # Setup for testing
+      record('Alert/Events', credentials, 'setup') do
+        @client = Hawkular::Alerts::AlertsClient.new(ALERTS_BASE, credentials, options)
+        json = IO.read('spec/integration/alert-resources/events-test-data.json')
+        hash = JSON.parse(json)
+        hash['events'].each do |event|
+          @client.create_event event['id'], event['category'], event['text'], event['extras']
```

Comment:
I'm using extras from because of this: https://github.com/hawkular/hawkular-client-ruby/blob/master/lib/hawkular/alerts/alerts_api.rb#L311
But i guess that for unifying the terminology it would be good to have those `extras` as named params. So the implementer wouldn't have to guess (or check the REST docs) to know what `extras` are available.